lib/scenic/adapters/postgres.rb

```diff
@@ -7,6 +7,17 @@ module Scenic
   # Scenic ships with a Postgres adapter only but can be extended with
   # additional adapters. The {Adapters::Postgres} adapter provides the
   # interface.
+  module Postgres
```

Comment:
Can we follow the same pattern I had followed with the connection and views objects? That is, extract these to `lib/scenic/adapters/postgres/errors.rb`, and require that file from here?